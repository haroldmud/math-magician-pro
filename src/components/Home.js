import Header from './Header';
import '../styles/Home.css';

const Home = () => (
  <section>
    <Header />
    <h1>Welcome to math magician</h1>
    <p>
      This Everyman’s Library edition–containing in one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes
      an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography.
      Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations.
      Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300.
      It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which
      Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original,
      renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
    </p>

    <p>
      Library edition–containing in one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes
      an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography.
      Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations.
      Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300.
      It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which
      Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original,
      renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
    </p>

    <p>
      In one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes
      an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography.
      Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations.
      Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300.
      It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which
      Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original,
      renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
    </p>
  </section>
);

export default Home;
