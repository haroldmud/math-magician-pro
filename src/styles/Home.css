section {
    padding-left: 2rem;
}

section p {
    width: 80%;
}