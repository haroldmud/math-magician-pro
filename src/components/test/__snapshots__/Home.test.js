import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import Home from '../../Home';
import '@testing-library/jest-dom';

describe('Test of the home page', () => {
  it('test the page rendering', () => {
    const homeTest = renderer
      .create(
        <Router>
          <Home />
        </Router>,
      )
      .toJSON();
    expect(homeTest).toMatchSnapshot();
  });
});
