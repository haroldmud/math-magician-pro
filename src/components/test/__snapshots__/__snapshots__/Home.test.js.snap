// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of the home page test the page rendering 1`] = `
<section>
  <header>
    <h1>
      Math Magician
    </h1>
    <nav>
      <a
        className="menu-item"
        href="#/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="menu-item"
        href="#/Calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
      <a
        className="menu-item"
        href="#/Quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </nav>
  </header>
  <h1>
    Welcome to math magician
  </h1>
  <p>
    This Everyman’s Library edition–containing in one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography. Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations. Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300. It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original, renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
  </p>
  <p>
    Library edition–containing in one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography. Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations. Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300. It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original, renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
  </p>
  <p>
    In one volume all three cantos, Inferno, Purgatorio, and Paradiso–includes an introduction by Nobel Prize—winning poet Eugenio Montale, a chronology, notes, and a bibliography. Also included are forty-two drawings selected from Botticelli's marvelous late-fifteenth-century series of illustrations. Translated in this edition by Allen Mandelbaum, The Divine Comedy begins in a shadowed forest on Good Friday in the year 1300. It proceeds on a journey that, in its intense recreation of the depths and the heights of human experience, has become the key with which Western civilization has sought to unlock the mystery of its own identity. Mandelbaum’s astonishingly Dantean translation, which captures so much of the life of the original, renders whole for us the masterpiece of that genius whom our greatest poets have recognized as a central model for all poets.
  </p>
</section>
`;
